src/tutorial/ja/01-Setup/e.md

```diff
@@ -0,0 +1,38 @@
+---
+out: Activator-Installation.html
+---
+
+  [Manual-Installation]: Manual-Installation.html
+
+Typesafe Activator (sbt を含む) のインストール 
+-------------------------------------------
+
+Typesafe Activator は `activator ui` と `activator new` という
+2つのコマンドを追加するカスタム版の sbt だ。
+つまり、`activator` は sbt の上位セットであると言える。
+
+Activator は [typesafe.com](http://typesafe.com/platform/getstarted) から取得できる。
+
+このガイドで `sbt ~test` というようなコマンドラインがあれば、
+`activator ~test` と打ち込めばそのまま動作するはずだ。
+Activator の「中の人」は sbt なので、全ての
+Activator プロジェクトは sbt で開くことができ、そのまた逆も成り立つ。
+
+Activator をダウンロードすると `activator` スクリプトと
+`activator-launch.jar` を含む。これは[手動インストール][Manual-Installation]で
+解説されている sbt スクリプトと sbt launcher JAR に相当する。
+以下が sbt の[手動インストール][Manual-Installation]との違いだ:
+
+- 引数なしで `activator` と打ち込むと `activator shell` モードか
+  `activator ui` モードに入るかを推論する。
+  コマンドラインプロンプトを強制したい場合は `activator shell` と打ち込む。
+- `activator new` を使うことで豊富な
+　[テンプレートのカタログ](https://typesafe.com/activator/templates)をもとにプロジェクトを新規作成することができる。
+  例えば、`play-scala` テンプレートを使うと Scala の [Play Framework](http://playframework.com) アプリを作れる。
```

Comment:
They seem prefer to use SSL. Might be better to have link to https://playframework.com?